courses/machine_learning/deepdive2/text_classification/solutions/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,1392 @@
+{
```

Comment:
<p>Consider ordering the text layers to match the layers in the model e.g. switch dense and dropout</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1412/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>